examples/example_run_simulation_with_economics.py

```diff
@@ -16,25 +16,14 @@
 from adapter_covid19.datasources import Reader, RegionSectorAgeDataSource
 from adapter_covid19.corporate_bankruptcy import CorporateBankruptcyModel
 from adapter_covid19.economics import Economics
-from adapter_covid19.gdp import LinearGdpModel, SupplyDemandGdpModel
+from adapter_covid19.gdp import PiecewiseLinearCobbDouglasGdpModel
```

Comment:
Running this example breaks with 

```
File "example_run_simulation_with_economics.py", line 461, in main
pb_model = PersonalBankruptcyModel(**example_parameters["personal_insolvency"])
TypeError: __init__() got an unexpected keyword argument 'saving'
```

Also, is there a reason why Personal Insolvencies are missing from the visualisation notebook?

Reply:
Re Personal Insolvencies: The primary output of the personal model is now household expenditure. The model reduces household expenditure before households become insolvent, so that personal insolvencies are very rare. We have thus omitted the chart as it is not informative.

Reply:
Deleted `example_run_simulation_with_economics` as it's no longer the best entrypoint to generating visualisations. `src/adapter_covid19/README.md` specifies the way the model should be run.